### PR TITLE
Parse cron variables

### PR DESCRIPTION
**Pull request by gefd**
This adds support for parsing and adding variables (such as MAILTO=email@host) in a crontab. It adds an additional Variable type with name and value attributes that can be added to, and retrieved from a Cronjob instance

### CODE REVIEW

File: src/Crontab/Variable.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * @package Crontab
+ */
+
+namespace Crontab;
+
+class Variable
+{
+	/**
```
Comment (blaugueux) [inline src/Crontab/Variable.php]: Please use a 4 spaces indent instead of tab.

**Comment by blaugueux** [comment]
Good job 👍 
Please fix the CS and squash your commits si i can merge your PR.

**Comment by blaugueux** [comment]
Thanks for your work :)

**Comment by gefd** [comment]
Thanks for merging the changes :)

**Pull request merged**